src/lib/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Spacer } from "@artsy/palette"
+import { CommercialButtons_artwork } from "__generated__/CommercialButtons_artwork.graphql"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import { BidButtonFragmentContainer as BidButton } from "./BidButton"
+import { BuyNowButtonFragmentContainer as BuyNowButton } from "./BuyNowButton"
+import { MakeOfferButtonFragmentContainer as MakeOfferButton } from "./MakeOfferButton"
+
+export interface CommercialButtonProps {
+  artwork: CommercialButtons_artwork
+  relay: RelayProp
+  // EditionSetID is passed down from the edition selected by the user
+  editionSetID?: string
+}
+
+export class CommercialButtons extends React.Component<CommercialButtonProps> {
+  handleInquiry = () => {
+    SwitchBoard.presentNavigationViewController(this, `/inquiry/${this.props.artwork.slug}`)
+  }
+
+  renderButtons = () => {
+    const { artwork } = this.props
+    const { isAcquireable, isOfferable, isInquireable, isInAuction } = artwork
+
+    if (isInAuction && artwork.sale && !artwork.sale.isClosed) {
```

Comment:
not a big deal but these two latter conditions are already taken care of in the bid button render method, so you could remove these checks and also the sale.isClosed selection from this fragment.